### PR TITLE
Statement cache not jmx

### PR DESCRIPTION
**Pull request by tjoneslo**
Remove the statement cache processing from the JMX as part of the remove all JMX interfaces from the SQL Layer server. 

Use `ALTER SERVER SET statementCacheCapacity to <N>` to set the size of of the statement cache. Use `SHOW statementCacheCapacity` to view the current size. Use `ALTER SERVER SET resetStatementCache TO TRUE` to clear all the statement caches.

I've left the statementCache hit/miss statistics and the access in `PostgresServer` but they are otherwise unused. They are (sort of) duplicated by the `I_S.server_statistics_summary` table `query_from_cache` (hit) and `query_count` columns. 

Because of the reliance of the statement cache testing now on the I_S tables, the `test-statement-cache.yaml` has been moved to an AAS-FTS test.

### CODE REVIEW

File: fdb-sql-layer-pg/src/main/java/com/foundationdb/sql/pg/PostgresServerConnection.java [hunk 1]
```diff
@@ -776,7 +776,8 @@ protected void processParse() throws IOException {
         for (int i = 0; i < nparams; i++)
             paramTypes[i] = messenger.readInt();
         sessionMonitor.startStatement(sql, stmtName);
-        logger.debug("Parse: {} = {} with types {}", stmtName, sql, paramTypes);
+        if (logger.isDebugEnabled())
+            logger.debug("Parse: " + stmtName + " = {} with types {}", sql, paramTypes);
```
Comment (ScottDugas) [inline fdb-sql-layer-pg/src/main/java/com/foundationdb/sql/pg/PostgresServerConnection.java]: Why did you change this? Was there a bug with the formatting version?

**Comment by nathanlws** [comment]
I think everything else on PostgresMXBean is exposed through SQL (via ALTER or i_s tables). Remove it altogether?

File: fdb-sql-layer-pg/src/main/java/com/foundationdb/sql/pg/PostgresServerConnection.java (hunk 1 above)
Comment (tjoneslo) [inline fdb-sql-layer-pg/src/main/java/com/foundationdb/sql/pg/PostgresServerConnection.java]: It was a complaint from Eclipse, I've reverted it.

**Comment by tjoneslo** [comment]
I had not verified that everything in MXBean is exposed elsewhere. I'll do that and update places, the delete the whole interface.

**Comment by tjoneslo** [comment]
PostgresMXBean deleted, and other instances.

**Comment by nathanlws** [comment]
Thanks.

Approve

>

**Comment by MMcM** [comment]
Approve

**Comment by nathanlws** [comment]
Unrelated.

Approve
